atlasdb-config/src/main/java/com/palantir/atlasdb/config/TimeLockClientConfig.java

```diff
@@ -27,4 +30,14 @@
     public abstract String client();
 
     public abstract ServerListConfig serversList();
+
+    public ServerListConfig toNamespacedServerList() {
```

Comment:
Yep, this should be called precisely twice over the lifetime of a TimeLockClientConfig, so the savings are minimal. I didn't know this feature of immutables existed though - thanks for the introduction 🙂